README.md

```diff
@@ -12,37 +12,33 @@ The main functionalities
 
 - Displays all the important laws that a consumer must be aware of
 
+# Screenshots
 
-<hr>  
-
+## Login & Register Page. 📟
```

Comment:
Hey the idea of adding emojis is good.
Could you add different relevant emoji for each heading
Change the heading size ```from ## Login & Register Page. to #### Login & Register Page.```

Reply:
Yeah sure I can do it.